libraries/AP_HAL_ChibiOS/hwdef/CUAV-7-Nano/README.md

```diff
@@ -0,0 +1,80 @@
+# CUAV-7-Nano Flight Controller
+
+The CUAV-7-Nano flight controller produced by [CUAV](https://www.cuav.net).
+
+## Features
+
+- STM32H753 microcontroller
+- 2 IMUs: IIM42652 and BMI088
+- builtin IST8310 magnetometer
+- 2 barometers: BMP581 and ICP20100
+- microSD card slot
+- USB-TypeC port
+- 1 ETH network interface
+- 5 UARTs plus USB
+- 14 PWM outputs
+- 3 I2C ports
+- 3 CAN ports (two of which share a CAN bus and one is an independent CAN bus)
+- Analog RSSI input
+- 3.3V/5V configurable PWM ouput voltage
+
+## Pinout
+
+![CUAV-7-Nano_interface_definition.png](CUAV-7-Nano-pinout.png)
+
+## UART Mapping
+
+- SERIAL0 -> USB
+- SERIAL1 -> UART7 (TELEM1)
+- SERIAL2 -> UART5 (TELEM2)
+- SERIAL3 -> USART1 (GPS&SAFETY)
+- SERIAL4 -> UART8 (GPS2)
+- SERIAL7 -> USART3 (FMU DEBUG)
+
+The TELEM1 and TELEM2 ports have RTS/CTS pins, the other UARTs do not have RTS/CTS.
```

Comment:
```suggestion
The TELEM1 and TELEM2 ports have RTS/CTS pins, the other UARTs do not have RTS/CTS. All have full DMA capability. SERIAL5 and SERIAL6 parameters appear but do not affect anything.
```

Reply:
still needs this change

Reply:
Sorry...I had modified it in the latest force-push. Could you check for the newer commit?

Reply:
you pushed and the change is not there